split-webapp/split/src/components/TransactionList.js

```diff
@@ -15,38 +21,67 @@ function ListItemLink(props) {
   return <ListItem button component="a" {...props} />;
 }
 
+function mapDataToElements(bills) {
+  return bills.map(bill => (
+    <ExpansionPanel>
+      <ExpansionPanelSummary>
+        <Typography variant="h5">
+          {bill.title}, ${bill.total}
+        </Typography>
+      </ExpansionPanelSummary>
+      <ExpansionPanelDetails>
+        {bill.payments.map(payment => {
+          const label = `${payment.from} owes ${payment.to} $${payment.amount}`;
+          return (
+            <div key={payment.id}>
+              <FormControlLabel
+                aria-label="Acknowledge"
+                onClick={event => event.stopPropagation()}
+                onFocus={event => event.stopPropagation()}
+                control={<Checkbox checked={payment.is_paid} />}
+                label={label}
+              />
+            </div>
+          );
+        })}
+      </ExpansionPanelDetails>
+    </ExpansionPanel>
+  ));
+}
+
 class TransactionList extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
       // If you're using state in component remove the es-lint line below, (else make it a functional component)
       // eslint-disable-next-line react/no-unused-state
-      transaction: {
-        users: [],
-        cost: 0,
-        payed: ""
-      }
+      bills: []
     };
   }
 
   componentDidMount() {
-    fetch("api/adress/goes/here")
+    //  actual address http://0.0.0.0:1234/api/bill_data/get_bill
+    fetch("https://jake-good.free.beeceptor.com/my/api/path") // temp
       .then(res => {
         return res.json();
       })
       .then(data => {
         // make a mapping of bills to list items here and render it below
-        console.log(data); // Temporary code (Adjust as required)
+        this.setState({
+          bills: data
+        });
       })
       .catch(err => {
-        alert(err);
+        alert.err(err);
```

Comment:
Should ` alert.err(err)` be `alert(err)` ? Just a thought, maybe we should `console.log()` this error for now instead, as having a error pop up might be confusing/strange for a user to see if the promise fails? 

Another alternative is that we make the error message more user friendly, alongside `console.log()` (for debugging purposes)  - This I guess is more a design decision though

Reply:
Removing the `alert(err)` would also prevent/fix issue #72

Reply:
Ok makes sense will fix that 👍